include/aspect/utilities.h

```diff
@@ -192,6 +192,21 @@ namespace aspect
                                                       double phi );   //longitude (radians)
 
     /**
+     * A struct to enable numerical output with a comma as thousands separator
+     */
+    struct ThousandSep : std::numpunct<char>
+    {
+      char do_thousands_sep() const
```

Comment:
The function is `virtual` in the base class. To make reading easier, mark it `virtual` here as well.